Traceback (most recent call last):
  File "snippet.py", line 104, in <module>
    main(sys.argv[1], sys.argv[2])
IndexError: list index out of range
